src/lib/game/world/map.js

```diff
@@ -86,6 +92,10 @@ class Map extends THREE.Group {
           }
 
           this.add(m2);
+
+          if (this.world !== null && m2.billboards.length > 0) {
+            this.world.addBillboardedM2(m2);
+          }
```

Comment:
Phasing is a gimmick, basically. In the parent world map, certain areas will have replacement maps that cover small subregions. From what I understand, phased creatures are then moved around the replacement map, rather than the parent map.